docs/src/getting-started/index.md

```diff
@@ -18,19 +18,22 @@ were a local Core Lightning node.
 ## Authentication
 
 All communication channels in Greenlight are authenticated and
-encrypted via mTLS (mutual Transport Layer Security), similar to HTTPS
-in the browser. Each client receives its own identity in the form of a
-private key and matching certificate, which can then be used to
-authenticate and encrypt communication when talking with Greenlight.
+encrypted via mTLS (mutual Transport Layer Security). Each client
+receives its own identity in the form of a private key and matching
+certificate, which can then be used to authenticate and encrypt
+communication when talking with Greenlight.
 
-In this guide we will be using two identities:
+This guide uses two types of identities
 
- - `nobody`: An identity that is shipped with the library, used to
-   communicate with services that don't require authentication.
- - A Device Identity: A unique identity used by the application to
-   authenticate with the Greenlight. The private key is generated and
-   kept locally, and certified by Greenlight to belong to a given
-   node.
+ - Developer identities can be used to register or recover Greenlight nodes.
+ - Device identities are used by applications to authenticate to single Greenlight
```

Comment:
I have been struggling with the wording here and think there are multiple options

- `user identities` is somewhat misleading because a single user could own multiple of these identities. E.g: one for each device
- `device identities` is somewhat confusing because both the developer certificate and this one could be on the same device
- `client identities` is probably a bit to vague. 

I think all of these options can work but we have to ensure we use consistent naming across all the docs

Reply:
I do think `device_identity` is preferred here.

Our code-examples will refer to a `device_key` and `device_cert` because this is what the _Scheduler_ returns on `register` or `recover`. 

If we want to have consistent naming in both the text and code-examples using `device`-identity is the only option.